### PR TITLE
[IMP] pos_self_order: merging mobile and kiosk

### PR DESCRIPTION
**Pull request by davidmonnom**
Previously, the self-order code and parameters were split into two parts
mobile and kiosk.

Now kiosk and mobile have been merged with each other to simplify
maintainability.

### CODE REVIEW

**Comment by davidmonnom** [comment]
@odoo/rd-security 

Hello, can you add security override ? We just change notification message on bus.

image

Thanks :)

File: addons/pos_self_order/__manifest__.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 {
     "name": "POS Self Order",
-    'version': '1.0.1',
+    'version': '1.0',
```
Comment (KangOl) [inline addons/pos_self_order/__manifest__.py:4]: Better increase the version number.
```suggestion
    'version': '1.1',
```

File: addons/point_of_sale/views/pos_session_view.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
             <form string="Point of Sale Session" create="0" edit="0">
                 <header>
                     <button name="open_frontend_cb" type="object" string="Continue Selling"
-                        invisible="rescue or state not in ['opening_control', 'opened'] or config_id.self_order_kiosk"/>
+                        invisible="rescue or state not in ['opening_control', 'opened'] or config_id.self_ordering_mode == 'kiosk'"/>
```
Comment (KangOl) [inline addons/point_of_sale/views/pos_session_view.xml:10]: You have a dependency issue, here. The field `self_ordering_mode` is not defined in the `point_of_sale` module.

Comment (davidmonnom) [reply addons/point_of_sale/views/pos_session_view.xml:10]: Good catch thx, I will fix that !

File: addons/pos_self_order/models/pos_config.py [hunk 3]
```diff
@@ -41,100 +32,73 @@ def _self_order_default_user(self):
...
         default="counter",
         help="Choose the kiosk mode",
         required=True,
     )
-    self_order_kiosk_available_language_ids = fields.Many2many(
-        "res.lang",
-        string="Available Languages",
-        help="Languages available for the kiosk mode",
-        default=_self_order_kiosk_default_languages,
+    self_ordering_alternative_fp_id = fields.Many2one(
```
Comment (KangOl) [inline addons/pos_self_order/models/pos_config.py:56]: Already defined like 37

**Comment by caburj** [comment]
robodoo r+

**Pull request closed without merging**